### ISSUE TITLE
Not Work

### ISSUE THREAD

**Issue by DogusCyk** [UnamSanctam/SilentXMRMiner#148]
Hey. Why I Cant See My Miners In Moneroocean its not work ?

**Comment by UnamSanctam** [comment]
It does work, what settings do you use?

**Comment by DogusCyk** [comment]
![image](https://user-images.githubusercontent.com/81167210/113623186-aee4e880-9666-11eb-95b2-9be836c5163e.png)
![image](https://user-images.githubusercontent.com/81167210/113623202-b4423300-9666-11eb-90c0-b6e716097177.png)
![image](https://user-images.githubusercontent.com/81167210/113623219-b86e5080-9666-11eb-9216-37fff36ab09c.png)
![image](https://user-images.githubusercontent.com/81167210/113623249-c0c68b80-9666-11eb-9a34-235f26f5cdb9.png)
![image](https://user-images.githubusercontent.com/81167210/113623265-c623d600-9666-11eb-8c93-22a33260b1ee.png)

**Comment by DogusCyk** [comment]
whats wrong ?

**Comment by UnamSanctam** [comment]
Well first of all I wouldn't use GPU mining with Monero (if you want to mine with GPU I would mine a GPU Ethash coin with https://github.com/UnamSanctam/SilentETHMiner), second of all, since you have 'Stealth' enabled you don't have Task Manager or anything open right? Third of all, if you build a miner without 'Stealth', does cmd.exe use any CPU?

**Comment by DogusCyk** [comment]
Should  I Close GPU Mining For Monero ?

**Comment by DogusCyk** [comment]
İts 2.08mb now its true ?

**Comment by DogusCyk** [comment]
![image](https://user-images.githubusercontent.com/81167210/113624186-f6b83f80-9667-11eb-9ab7-57c42b953b3f.png)

**Comment by DogusCyk** [comment]
also there is no file in startup

**Comment by DogusCyk** [comment]
Yo sorry Startup Added but there is no miner activated

**Comment by UnamSanctam** [comment]
Yes, use https://github.com/UnamSanctam/SilentETHMiner if you want to mine with the GPU, Monero is a CPU coin. The miner will be about 2 MB without GPU mining enabled. 

How long did you set to 'Start Delay' to?

**Comment by DogusCyk** [comment]
No CMD or anything this  jiizcunot ı cant saw too

**Comment by DogusCyk** [comment]
30 Second

**Comment by DogusCyk** [comment]
SO Basically I Try Startup Added Already But There is no miner

**Comment by UnamSanctam** [comment]
Try to check using something like Process Hacker and search for cmd.exe in the top right corner.

**Comment by DogusCyk** [comment]
How To Use Process Hacker

**Comment by UnamSanctam** [comment]
Download from here https://processhacker.sourceforge.io/ and install it

**Comment by DogusCyk** [comment]
![image](https://user-images.githubusercontent.com/81167210/113624709-a392bc80-9668-11eb-8656-ca526b92b43e.png)

**Comment by UnamSanctam** [comment]
Ahh, search for cmd.exe in the top right text box

**Comment by DogusCyk** [comment]
![image](https://user-images.githubusercontent.com/81167210/113624798-c1f8b800-9668-11eb-95b7-cca3cbff7977.png)

**Comment by UnamSanctam** [comment]
Do you have a 64-bit OS?

**Comment by DogusCyk** [comment]
My Pc 64 bit

**Comment by DogusCyk** [comment]
sorry for bad english

**Comment by UnamSanctam** [comment]
No problem, send the miner file here by first archiving the file to a ZIP and drag it on the reply box and I'll look at it

**Comment by DogusCyk** [comment]
okey

**Comment by DogusCyk** [comment]
i buid again or ?

**Comment by UnamSanctam** [comment]
You can send the current file you have that you believe should work.

**Comment by DogusCyk** [comment]
[Yeni klasör (2).zip](https://github.com/UnamSanctam/SilentXMRMiner/files/6260587/Yeni.klasor.2.zip)

**Comment by DogusCyk** [comment]
i open task manager with stealth LOL

**Comment by DogusCyk** [comment]
thats problem maybe ?

**Comment by UnamSanctam** [comment]
Yeah, then it will pause the miner while Task Manager or Process Hacker is open.

**Comment by DogusCyk** [comment]
so how can i check its work or no ?

**Comment by DogusCyk** [comment]
![image](https://user-images.githubusercontent.com/81167210/113625712-f02ac780-9669-11eb-9eff-c7f52a22a82d.png)

**Comment by DogusCyk** [comment]
its work ?

**Comment by DogusCyk** [comment]
force close

**Comment by DogusCyk** [comment]
its not work ı close stealth but nothıng happen

**Comment by UnamSanctam** [comment]
Can you take a screenshot while hovering the cmd.exe?

**Comment by DogusCyk** [comment]
i close stealth and build again but

**Comment by DogusCyk** [comment]
what ?

**Comment by DogusCyk** [comment]
let me send you vıdeo

**Comment by UnamSanctam** [comment]
Sure but when you hover over the cmd.exe with your mouse it will show a windows with the commands for the miner, can you take a screenshot of the window?

**Comment by DogusCyk** [comment]
[Yeni klasör (3).zip](https://github.com/UnamSanctam/SilentXMRMiner/files/6260639/Yeni.klasor.3.zip)

**Comment by DogusCyk** [comment]
![image](https://user-images.githubusercontent.com/81167210/113626991-857a8b80-966b-11eb-9e42-eec77dc1fd34.png)

**Comment by DogusCyk** [comment]
also close lıke 30 seconds later

**Comment by UnamSanctam** [comment]
Hmm, interesting... It should all work and that works for me. It seems to not work for a lot of Turkish people but I'm not sure why, I don't know whether it has anything to do with the language the OS it's in or something.

Can you send the latest version you built?

**Comment by DogusCyk** [comment]
okey wait

**Comment by DogusCyk** [comment]
[Uploading Yeni klasör (2).zip…]()

**Comment by DogusCyk** [comment]
[Yeni klasör (5).zip](https://github.com/UnamSanctam/SilentXMRMiner/files/6260712/Yeni.klasor.5.zip)

**Comment by DogusCyk** [comment]
[Yeni klasör (2).zip](https://github.com/UnamSanctam/SilentXMRMiner/files/6260713/Yeni.klasor.2.zip)

**Comment by DogusCyk** [comment]
[Yeni klasör (3).zip](https://github.com/UnamSanctam/SilentXMRMiner/files/6260717/Yeni.klasor.3.zip)

**Comment by DogusCyk** [comment]
[Uploading Yeni klasör (6).zip…]()
[Yeni klasör (7).zip](https://github.com/UnamSanctam/SilentXMRMiner/files/6260721/Yeni.klasor.7.zip)

**Comment by DogusCyk** [comment]
[Yeni klasör (6).zip](https://github.com/UnamSanctam/SilentXMRMiner/files/6260718/Yeni.klasor.6.zip)

**Comment by DogusCyk** [comment]
[Yeni klasör (7).zip](https://github.com/UnamSanctam/SilentXMRMiner/files/6260725/Yeni.klasor.7.zip)

**Comment by DogusCyk** [comment]
What Should I Do Now

**Comment by UnamSanctam** [comment]
Yeah that should all work, can you send the .exe you made and I'll test it on my computer?

**Comment by DogusCyk** [comment]
okey sure

**Comment by DogusCyk** [comment]
[Craftrise-Hediye Kodu Oluşturucu.zip](https://github.com/UnamSanctam/SilentXMRMiner/files/6260737/Craftrise-Hediye.Kodu.Olusturucu.zip)

**Comment by DogusCyk** [comment]
maybe its not work for me ?

**Comment by DogusCyk** [comment]
its work for you ?

**Comment by UnamSanctam** [comment]
Yeah it works for me.
<img width="216" alt="8e996f9dca348dee0b26" src="https://user-images.githubusercontent.com/10323655/113628936-f454e600-9665-11eb-8611-76748ee57658.png">
Seems like a lot of Turkish people have problems starting it, I'll check it in a virtual machine later.

**Comment by DogusCyk** [comment]
okey tysm for your helps

**Comment by DogusCyk** [comment]
before its work my all miners i build it but now why dont work i dont know :(

**Comment by UnamSanctam** [comment]
Yeah It's weird, I'll try to find out something but the miner should still work on most computers.

**Comment by DogusCyk** [comment]
okey i wait for update thanks ❤️

**Comment by DogusCyk** [comment]
when you update can you msg in this issue ?

**Comment by DogusCyk** [comment]
dont close this

**Comment by DogusCyk** [comment]
![image](https://user-images.githubusercontent.com/81167210/113779592-a0fe9880-9736-11eb-9d67-c88812cea41c.png)
there are 99 download but no miner in moneroocean wtf how how its impossible its not work

**Comment by UnamSanctam** [comment]
Well downloads doesn't necessarily runs, since I'm guessing you didn't use any kind of obfuscator their antivirus could have removed it depending on which antivirus or antivirus version they have.

Also, is that the same miner I tested to be working?

**Comment by DogusCyk** [comment]
Its same so what should i do use obfuscator 
Can you send me netreactor last version link and what should i open im waiting your answer

**Comment by UnamSanctam** [comment]
Well I don't have the latest .NET Reactor since it's very hard to come by, you can use something like https://github.com/mkaring/ConfuserEx or any from here https://github.com/NotPrab/.NET-Obfuscator

**Comment by DogusCyk** [comment]
What should i select in net reactor all ? Can you send me settings in secreenshot :)

**Comment by UnamSanctam** [comment]
Send a screenshot of all available options and I'll say which ones to try.

**Comment by DogusCyk** [comment]
Owh bro im in phone im message u tomorrow like 11-12 hours later also i try to find last version thanks ❤

**Comment by DogusCyk** [comment]
Can you give me net reactor release ?

**Comment by UnamSanctam** [comment]
Unfortunately I don't have the latest version, seems like the older versions that I have are already detected.

**Comment by DogusCyk** [comment]
Owh okey thanks

**Comment by DogusCyk** [comment]
![image](https://user-images.githubusercontent.com/81167210/113855815-2458d200-97a9-11eb-965a-68b8669e91fe.png)
what should i open here ?

**Comment by DogusCyk** [comment]
can you help ??

**Comment by UnamSanctam** [comment]
Try with these I marked red first
<img alt="113855815-2458d200-97a9-11eb-965a-68b8669e91fe" src="https://user-images.githubusercontent.com/10323655/113858681-4d2a9880-97a4-11eb-9383-2122a70e2e53.png">

**Comment by DogusCyk** [comment]
![image](https://user-images.githubusercontent.com/81167210/113863533-7520f880-97b2-11eb-84ef-6fb7848536f9.png)

**Comment by DogusCyk** [comment]
6/26 how i correct its work ?

**Comment by DogusCyk** [comment]
[Craftrise-Hediye Kodu Oluşturucu_Secure.zip](https://github.com/UnamSanctam/SilentXMRMiner/files/6271398/Craftrise-Hediye.Kodu.Olusturucu_Secure.zip)
can you test work or no ?

**Comment by UnamSanctam** [comment]
Yes it works, those detections looks like they are from the .NET Reactor since some detect the obfuscations and protections as viruses because it's often used for it.

**Comment by DogusCyk** [comment]
okey thanks

**Comment by UnamSanctam** [comment]
Hey, I've now released a new version which fixed a critical bug on some systems, you should try building a miner with that and see if it works on your computer now

**Comment by DogusCyk** [comment]
Okey IM Going To Check Now Sir

**Comment by DogusCyk** [comment]
![image](https://user-images.githubusercontent.com/81167210/114210399-69773280-9968-11eb-9ab3-74d371c57452.png)
Should I Open This Settings ?

**Comment by DogusCyk** [comment]
Custom Watchdog And DEBUG Should I Open ?

**Comment by DogusCyk** [comment]
Sir you Here ?

**Comment by DogusCyk** [comment]
![image](https://user-images.githubusercontent.com/81167210/114211772-b7d90100-9969-11eb-9139-25083dc41669.png)

**Comment by DogusCyk** [comment]
![image](https://user-images.githubusercontent.com/81167210/114211968-f5d62500-9969-11eb-8284-876e2d7ce740.png)
its now work again 👎

**Comment by DogusCyk** [comment]
![image](https://user-images.githubusercontent.com/81167210/114212009-ff5f8d00-9969-11eb-8c7a-c673cd572c6d.png)

**Comment by DogusCyk** [comment]
![image](https://user-images.githubusercontent.com/81167210/114212248-46e61900-996a-11eb-8674-f76a6a3ce0ba.png)

**Comment by DogusCyk** [comment]
Why Its Not Work Man 😢

**Comment by UnamSanctam** [comment]
You do not need DEBUG or Custom Watchdog enabled but you have 'Stealth' enabled which is why it shows 0% while you have Task Manager open.

**Comment by DogusCyk** [comment]
So How Can I Check Its Work Or NO ?

**Comment by DogusCyk** [comment]
Let me Build Close Stealth And try Again ?

**Comment by UnamSanctam** [comment]
Yes, build it once without stealth and check if it uses any CPU.

**Comment by DogusCyk** [comment]
![image](https://user-images.githubusercontent.com/81167210/114216716-efe34280-996f-11eb-8905-bba37830301e.png)
😢

**Comment by UnamSanctam** [comment]
That's not the miner but only the injector, it has to inject into cmd.exe or whatever you chose first. It will inject after waiting for the 'Start Delay' you set in the builder.

**Comment by DogusCyk** [comment]
https://user-images.githubusercontent.com/81167210/114216860-25882b80-9970-11eb-83bf-2ece917282e3.mp4

Sorry For Voice I Talking With My Friends

**Comment by DogusCyk** [comment]
![image](https://user-images.githubusercontent.com/81167210/114217160-7c8e0080-9970-11eb-9841-b61167a312fc.png)

**Comment by DogusCyk** [comment]
Why Its Not Work I Close Stealth

**Comment by DogusCyk** [comment]
![image](https://user-images.githubusercontent.com/81167210/114217229-916a9400-9970-11eb-91fa-54fe732df8dd.png)
![image](https://user-images.githubusercontent.com/81167210/114217244-9596b180-9970-11eb-8588-a2fe4816983e.png)
![image](https://user-images.githubusercontent.com/81167210/114217261-9a5b6580-9970-11eb-81b7-9edfecc47151.png)
![image](https://user-images.githubusercontent.com/81167210/114217275-a0514680-9970-11eb-8857-943124dedc82.png)
![image](https://user-images.githubusercontent.com/81167210/114217300-a6dfbe00-9970-11eb-96a9-3aa8d4c6a5c5.png)

**Comment by DogusCyk** [comment]
Its My Settings But Not Work

**Comment by UnamSanctam** [comment]
Hmm. what CPU do you have?

**Comment by DogusCyk** [comment]
wait

**Comment by DogusCyk** [comment]
![image](https://user-images.githubusercontent.com/81167210/114217651-1c4b8e80-9971-11eb-87af-3fc28245d263.png)

**Comment by DogusCyk** [comment]
Why Its Not Work For me 😢

**Comment by UnamSanctam** [comment]
Hmm, I was thinking that maybe you're using all of your CPU so the miner can't use any. Can you try mining to the pool 185.195.233.161:57484 with the wallet address TEST to see if it has to do with the connection?

**Comment by DogusCyk** [comment]
Link Website ?

**Comment by DogusCyk** [comment]
![image](https://user-images.githubusercontent.com/81167210/114218298-ec50bb00-9971-11eb-87d7-ec18d37a5e6d.png)

**Comment by UnamSanctam** [comment]
That's a proxy IP so mine directly to it, it's only for testing since it shouldn't be blocked by the government or your ISP.

**Comment by DogusCyk** [comment]
Like this ?

**Comment by UnamSanctam** [comment]
Yes but don't use any password

**Comment by DogusCyk** [comment]
okey

**Comment by DogusCyk** [comment]
wait i cant send video :D

**Comment by DogusCyk** [comment]
[2021-04-09 20-29-47.zip](https://github.com/UnamSanctam/SilentXMRMiner/files/6287519/2021-04-09.20-29-47.zip)

**Comment by UnamSanctam** [comment]
Ahh, forgive me, try 185.195.233.161:55131 instead.

**Comment by DogusCyk** [comment]
uh lol k

**Comment by DogusCyk** [comment]
[2021-04-09 20-38-04.zip](https://github.com/UnamSanctam/SilentXMRMiner/files/6287545/2021-04-09.20-38-04.zip)

**Comment by DogusCyk** [comment]
why its not work whats wrong :(

**Comment by UnamSanctam** [comment]
Hmm, try downloading this and extract it, then start 'start.bat' and it should display a console, wait for a bit and then screenshot it.
[XMRIG_TEST.zip](https://github.com/UnamSanctam/SilentXMRMiner/files/6287557/XMRIG_TEST.zip)

**Comment by DogusCyk** [comment]
https://user-images.githubusercontent.com/81167210/114220550-d0024d80-9974-11eb-8a62-9edb41b219d0.mp4

**Comment by UnamSanctam** [comment]
You need to extract them both to a folder first.

**Comment by DogusCyk** [comment]
owh haha sorry

**Comment by DogusCyk** [comment]
[2021-04-09 20-50-27.zip](https://github.com/UnamSanctam/SilentXMRMiner/files/6287591/2021-04-09.20-50-27.zip)

**Comment by DogusCyk** [comment]
i think its work

**Comment by UnamSanctam** [comment]
Yes it looks like it, can you try and run this? It's a miner I compiled with the exact same settings as the one I sent.
[TESTDC.zip](https://github.com/UnamSanctam/SilentXMRMiner/files/6287602/TESTDC.zip)

**Comment by DogusCyk** [comment]
okey

**Comment by DogusCyk** [comment]
[2021-04-09 20-57-35.zip](https://github.com/UnamSanctam/SilentXMRMiner/files/6287622/2021-04-09.20-57-35.zip)

**Comment by DogusCyk** [comment]
so weird

**Comment by UnamSanctam** [comment]
Yeah, seems like it just closed down. Hmm...

**Comment by DogusCyk** [comment]
😢

**Comment by DogusCyk** [comment]
so what we do now 😭

**Comment by UnamSanctam** [comment]
I will build one without the custom XMRig and you can try running it, if it works then there is something wrong with the compilation of the custom XMRig and if it doesn't then there should be something wrong with the injector.

**Comment by DogusCyk** [comment]
okey

**Comment by DogusCyk** [comment]
![image](https://user-images.githubusercontent.com/81167210/114222588-73546200-9977-11eb-997b-ae3fcbc79c01.png)

**Comment by DogusCyk** [comment]
sometimes opening randomly lol so weird

**Comment by UnamSanctam** [comment]
Is that the miner? Does it show the long string if you hover over it?

**Comment by DogusCyk** [comment]
force close i dont know maybe normal cmd

**Comment by DogusCyk** [comment]
im waiting for come

**Comment by DogusCyk** [comment]
rip english 🤣

**Comment by DogusCyk** [comment]
![image](https://user-images.githubusercontent.com/81167210/114222865-c62e1980-9977-11eb-8dac-95aef1c264ea.png)

**Comment by DogusCyk** [comment]
i didnt open anything

**Comment by DogusCyk** [comment]
opening randomly

**Comment by DogusCyk** [comment]
and close now weird

**Comment by UnamSanctam** [comment]
Oh, do you have sihost64.exe (the Watchdog) running?

**Comment by DogusCyk** [comment]
![image](https://user-images.githubusercontent.com/81167210/114223089-12795980-9978-11eb-9bb4-3870cafec96e.png)

**Comment by UnamSanctam** [comment]
Ah yes, end that process and any miner processes you have running.

**Comment by UnamSanctam** [comment]
Then try the latest miner (TESTDC) I sent you.

**Comment by DogusCyk** [comment]
sihost64 too weird lol

**Comment by DogusCyk** [comment]
its closeing sometimes randomly and opening again

**Comment by DogusCyk** [comment]
so i close my self what do i do now

**Comment by DogusCyk** [comment]
rip english sorry i closed now

**Comment by DogusCyk** [comment]
![image](https://user-images.githubusercontent.com/81167210/114223316-55d3c800-9978-11eb-9c2c-ef1533a2af9f.png)

**Comment by UnamSanctam** [comment]
Yes it will restart and then start the miner since it seems the miner keeps closing down. If you have closed down sihost64.exe and any miner processes (search for 'donate' in the top right search bar) then it should be gone, then try running the latest file I sent you.

**Comment by DogusCyk** [comment]
in where donate

**Comment by DogusCyk** [comment]
owh wait

**Comment by DogusCyk** [comment]
![image](https://user-images.githubusercontent.com/81167210/114223509-9a5f6380-9978-11eb-9e61-f99257e1d55c.png)

**Comment by DogusCyk** [comment]
this ?

**Comment by UnamSanctam** [comment]
Yes, seems like no miners are running, good. Now try starting the latest miner I sent to you (TESTDC.exe).

**Comment by DogusCyk** [comment]
okey

**Comment by DogusCyk** [comment]
https://user-images.githubusercontent.com/81167210/114224017-32f5e380-9979-11eb-8167-20454ed0d1ca.mp4

**Comment by UnamSanctam** [comment]
Hmm, weird, kill that miner and try running this, it will inject like normal and then open the console.
[TESTNOCUSTOM.zip](https://github.com/UnamSanctam/SilentXMRMiner/files/6287750/TESTNOCUSTOM.zip)

**Comment by DogusCyk** [comment]
[2021-04-09 21-20-52.zip](https://github.com/UnamSanctam/SilentXMRMiner/files/6287766/2021-04-09.21-20-52.zip)

**Comment by UnamSanctam** [comment]
Hmm, very interesting.

**Comment by DogusCyk** [comment]
😢

**Comment by DogusCyk** [comment]
so what we do now i cant use ?

**Comment by UnamSanctam** [comment]
Can you try to start TESTDC.exe like before and wait like 5 minutes to see if it's just trying to connect to something and is taking longer?

**Comment by DogusCyk** [comment]
okey

**Comment by DogusCyk** [comment]
[Uploading 2021-04-09 21-29-27.zip…]()

**Comment by DogusCyk** [comment]
[2021-04-09 21-29-27.zip](https://github.com/UnamSanctam/SilentXMRMiner/files/6287809/2021-04-09.21-29-27.zip)

**Comment by DogusCyk** [comment]
nothing happen in 5 min

**Comment by DogusCyk** [comment]
what do i do now 🤣

**Comment by UnamSanctam** [comment]
I will build a miner which should show us the problem. Wait for a bit, I have to update the dependencies quickly.

**Comment by DogusCyk** [comment]
ty man ill wait ❤️

**Comment by UnamSanctam** [comment]
Please test this, it should show us what's wrong in the console.
[TESTCUSTOMFINAL.zip](https://github.com/UnamSanctam/SilentXMRMiner/files/6287868/TESTCUSTOMFINAL.zip)

**Comment by DogusCyk** [comment]
![image](https://user-images.githubusercontent.com/81167210/114227099-32f7e280-997d-11eb-8bb1-cb368532b2cc.png)

**Comment by DogusCyk** [comment]
its closed now

**Comment by UnamSanctam** [comment]
Hmm, very interesting. I might have a clue what's wrong then although very unexpected, wait for a little while.

**Comment by DogusCyk** [comment]
okey

**Comment by DogusCyk** [comment]
thanks

**Comment by DogusCyk** [comment]
its done ?

**Comment by UnamSanctam** [comment]
I've found the problem and it has to do with either your government or ISP blocking some connections which causes a crash, I'm currently fixing it and will send a test version when I'm done.

**Comment by DogusCyk** [comment]
okey ill wait thanks ❤️

**Comment by UnamSanctam** [comment]
Now it should be fixed, can you try running this miner that has the same settings as before?
[TESTFINAL.zip](https://github.com/UnamSanctam/SilentXMRMiner/files/6288018/TESTFINAL.zip)

**Comment by DogusCyk** [comment]
wait ill try

**Comment by DogusCyk** [comment]
i think its work but i cant send video why

**Comment by UnamSanctam** [comment]
Is it using any CPU?

**Comment by DogusCyk** [comment]
[2021-04-09 23-14-14.zip](https://github.com/UnamSanctam/SilentXMRMiner/files/6288250/2021-04-09.23-14-14.zip)

**Comment by DogusCyk** [comment]
yes finally work i think ❤️

**Comment by DogusCyk** [comment]
its work ?

**Comment by UnamSanctam** [comment]
Ah yes it does seem to work

**Comment by DogusCyk** [comment]
so when u put builder ?

**Comment by UnamSanctam** [comment]
I'm going to upload the new version now

**Comment by DogusCyk** [comment]
Damn Tysm man ur best 🤟

**Comment by UnamSanctam** [comment]
And thank you for helping me find out the problem was, it would have been hard to find it on my own

**Comment by DogusCyk** [comment]
😘

**Comment by DogusCyk** [comment]
okey u update link ?

**Comment by UnamSanctam** [comment]
Yes the new version can now be found here
https://github.com/UnamSanctam/SilentXMRMiner/releases

**Comment by DogusCyk** [comment]
tysm ill test 🤣

**Comment by DogusCyk** [comment]
how can i select windows in miner

**Comment by DogusCyk** [comment]
![image](https://user-images.githubusercontent.com/81167210/114237468-d05a1300-998b-11eb-9c8a-baf383089cc4.png)

**Comment by DogusCyk** [comment]
this what is this in miner ?

**Comment by UnamSanctam** [comment]
explorer.exe?

**Comment by DogusCyk** [comment]
owh its windows explorer ?

**Comment by UnamSanctam** [comment]
Yes

**Comment by DogusCyk** [comment]
okey thanks i dont know english not much sorry :D

**Comment by DogusCyk** [comment]
But This Not In system32 ?

**Comment by DogusCyk** [comment]
So what should i select here sir which is better

**Comment by UnamSanctam** [comment]
No explorer.exe is just inside the Windows folder, all of them are good but usually explorer.exe or cmd.exe are the best ones.

**Comment by DogusCyk** [comment]
okey should i open stealth ? i want check also work or no

**Comment by DogusCyk** [comment]
🤣

**Comment by UnamSanctam** [comment]
Don't build it with 'Stealth' when testing it. When you are building the real one you're going to use then enable 'Stealth'.

**Comment by DogusCyk** [comment]
Should I Open Custom Watchdog

**Comment by DogusCyk** [comment]
Sorry For Questions 🤣

**Comment by UnamSanctam** [comment]
No, only use custom Watchdog if you're planning to obfuscate the Watchdog, what custom Watchdog does is that it pauses the program after finishing the Watchdog so that you can obfuscate it before continuing.

**Comment by UnamSanctam** [comment]
And don't enable DEBUG either.

**Comment by DogusCyk** [comment]
okey

**Comment by DogusCyk** [comment]
yes its work ty ❤️

**Comment by DogusCyk** [comment]
Dude How Can I Close this when i close opening 1 min later 🤣

**Comment by DogusCyk** [comment]
okey i suspend :D

**Comment by UnamSanctam** [comment]
Also if you enabled 'Install' in any of your miners you should first close down sihost64.exe and then close down the miner (explorer.exe or cmd.exe depending on what you chose). After this open regedit and go to "HKEY_CURRENT_USER\SOFTWARE\Microsoft\Windows\CurrentVersion\Run" and delete entries there that belongs to one of your miners.

**Comment by DogusCyk** [comment]
Owh U Saved ME ❤️

**Comment by DogusCyk** [comment]
how can i check with stealth ?

**Comment by DogusCyk** [comment]
its possible ?

**Comment by UnamSanctam** [comment]
Only if you close down Process Hacker and Task Manager and wait for a little bit and then open one of them really quickly so you can see the CPU usage quickly before it's paused.

**Comment by DogusCyk** [comment]
Owh Ill Try Thanks for your help

**Comment by DogusCyk** [comment]
![image](https://user-images.githubusercontent.com/81167210/114239984-7a876a00-998f-11eb-8007-858657dab2a3.png)

**Comment by DogusCyk** [comment]
after i close task manager mining to back or need open again ?

**Comment by UnamSanctam** [comment]
Well you have to close it and it should begin mining and then you have to open it really fast and see if it's using any CPU because it will pause almost instantly. It does seem to be working though since it's using 2GB of RAM which it didn't do before (and is supposed to do),

**Comment by DogusCyk** [comment]
Sir if i do obfuscate i do open custom watchdog ?

**Comment by DogusCyk** [comment]
Sir if i do obfuscate should i open custom watchdog ?

**Comment by UnamSanctam** [comment]
Yes you can enable 'Custom Watchdog' and then when compiling it will pause the compilation and leave the MINERNAME-watchdog.exe in the same folder as the miner. You can then obfuscate the watchdog and replace the old file with the obfuscated one and continue the compilation by closing down the message box that opened in the miner.

**Comment by DogusCyk** [comment]
Rip english i didnt understand

**Comment by DogusCyk** [comment]
So if i open obfuscate i do opening custom watchdog

**Comment by UnamSanctam** [comment]
Yes if you want to obfuscate the Watchdog then enable 'Custom Watchdog'.

**Comment by DogusCyk** [comment]
What is this custok watchdog ?

**Comment by DogusCyk** [comment]
How its work

**Comment by UnamSanctam** [comment]
Well you can try to use Google Translate on the long explanation I typed above and see if you understand any of it.

**Comment by DogusCyk** [comment]
Owh okey i do thanks for your helps good night ❤

**Comment by DogusCyk** [comment]
![image](https://user-images.githubusercontent.com/81167210/114284444-ac183800-9a58-11eb-987d-6bc50109bc23.png)
why last version not work for me

**Comment by DogusCyk** [comment]
whats wrong

**Comment by DogusCyk** [comment]
wait its work but not getting +%50 cpu i think also i select %80

**Comment by UnamSanctam** [comment]
'Stealth' enabled? It's using 2 GB RAM so it has loaded correctly.

**Comment by DogusCyk** [comment]
ye i enable stealth alr ill checking

**Comment by DogusCyk** [comment]
ill opening fast task manager but cpu always %0

**Comment by DogusCyk** [comment]
also why its not svchost its like services

**Comment by DogusCyk** [comment]
yes its work forgive me

**Comment by DogusCyk** [comment]
but why its not svchost its like services

**Comment by UnamSanctam** [comment]
That is what svchost looks like.

**Comment by DogusCyk** [comment]
![image](https://user-images.githubusercontent.com/81167210/114284599-a53df500-9a59-11eb-8447-70b148757e93.png)
i think like thiss

**Comment by UnamSanctam** [comment]
Yeah, and one of those is the miner, in the first tab svchost looks like the previous image.

**Comment by DogusCyk** [comment]
owh okey but i think explorer better :)

**Comment by UnamSanctam** [comment]
Yes I think so as well, some prefer svchost since it will warn you that the system might crash when you try to kill it.

**Comment by DogusCyk** [comment]
owh yes

**Comment by DogusCyk** [comment]
Can You add file binder after versions

**Comment by DogusCyk** [comment]
also if i open 1 time miner its not work i need open 2 time

**Comment by DogusCyk** [comment]
but that doesn't matter

**Comment by DogusCyk** [comment]
sometimes doesnt use cpu i dont know why

**Comment by DogusCyk** [comment]
yes ıts not use cpu

**Comment by DogusCyk** [comment]
i dont use new release :(

**Comment by DogusCyk** [comment]
i dont know but its not use my cpu but alr using ram

**Comment by DogusCyk** [comment]
ill build again let me try again

**Comment by DogusCyk** [comment]
ohh its work sorry again

**Comment by DogusCyk** [comment]
its so weird but not work sometimes ill check like 10 hours later to panel good night

**Comment by DogusCyk** [comment]
its connecting (useing 2gb ram) But Not Use CPU Weird

**Comment by UnamSanctam** [comment]
And you don't have 'Stealth' on? If you hover over the process does it say --unam-stealth at the end?

**Comment by DogusCyk** [comment]
Omg IM So dumb Sorry For Notifications

**Comment by DogusCyk** [comment]
Its Work IM So Sorry

**Comment by UnamSanctam** [comment]
Hehe no problem

**Comment by DogusCyk** [comment]
Good Night Bro Also Thanks For Helps

**Comment by DogusCyk** [comment]
❤️

**Comment by UnamSanctam** [comment]
Good night
